doc/ref_model/chapters/chapter03.md

```diff
@@ -285,7 +285,29 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 ### 3.8.1 Types of accelerators
 
 <a name="3.8.2"></a>
-### 3.8.2 Levels of accelerator consumption
+### 3.8.2 Infrastructure and Application Level Acceleration
+
+Figure 3-8 gives examples for Hardware Accelerators in [Sample reference model realization](#3.7) diagram.
+
+<p align="center"><img src="../figures/ch03-hardware-acceleration-in-rm-realization-diagram.png" alt="Hardware Acceleration in RM Realization Diagram" Title="Hardware Acceleration in RM Realization Diagram" width="65%"/></p>
```

Comment:
Please add the figure in .png and in editable version.

Reply:
> Please add the figure in .png and in editable version.

@CsatariGergely : It is in #2162 . (Sorry, then I didn't know how in web UI to add it together with text, now I know.)